tests/cpp/test_matmul.cpp

```diff
@@ -449,6 +449,35 @@ TEST_P(MatmulTestWithLayout, AmpereMatmulPipelineGmem) {
   }
 }
 
+// Check that mma op is not predicated.
+class PredicateChecker : public kir::IrVisitor {
```

Comment:
I suppose instead of using this we could possibly use `PredicatedChecker::isPredicated` instead. I kept it here to mirror the check in the `AmpereSwizzle` test.